blockdag/mining.go

```diff
@@ -34,25 +37,37 @@ func (dag *BlockDAG) BlockForMining(transactions []*util.Tx) (*wire.MsgBlock, er
 		msgBlock.AddTransaction(tx.MsgTx())
 	}
 
-	utxoWithTransactions, err := dag.UTXOSet().WithTransactions(msgBlock.Transactions, UnacceptedBlueScore, false)
+	ms, err := dag.NextBlockMultiset(transactions)
```

Comment:
Please use a longer name than ms.
In other places where the multiset is the key thing, I didn't mind too much(although still better to call it multiSet), but here this is more important because ms is no longer the center of all things, and the word ms can mean a ton of things